scripts/dts/extract/globals.py

```diff
@@ -331,6 +331,44 @@ def build_cell_array(prop_array):
 
     return ret_array
 
+def gpio_map_lookup(cell_parent, gpio_index):
+    index = 0
+    map_array = []
+
+    # Aim of this function is to return a gpio-controller phandle
+    # from the object provided (cell_parent: gpio-map) thanks to the
```

Comment:
Does the `:` mean that `cell_parent` is a `gpio-map`? Might be clearer to write it out in that case ("cell_parent, which is a gpio-map").

"From the object provided" is a bit confusing since the function takes two arguments.

Reply:
Reworked